src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2

```diff
@@ -74,6 +74,7 @@ resource "aws_dlm_lifecycle_policy" "data_volume_backups" {
 
     target_tags = {
       GroupDetail = "couchdb2:data"
+      Group = "elasticsearch"
```

Comment:
Just checking, does terraform / AWS interpret this as "if `GroupDetail = "couchdb2:data"` AND `Group = "elasticsearch"`" or "if `GroupDetail = "couchdb2:data"` OR `Group = "elasticsearch"`"

Reply:
confirmed that the condition here matches volumes that have either tag